go.mod

```diff
@@ -8,6 +8,7 @@ replace github.com/coreos/etcd => github.com/coreos/etcd v3.3.24+incompatible
 require (
 	github.com/ONSdigital/dp-component-test v0.6.5
 	github.com/ONSdigital/dp-healthcheck v1.3.0
+	github.com/ONSdigital/dp-kafka/v2 v2.4.4
```

Comment:
Why has this been changed to an older version of dp-kafka?

Reply:
Its not changed but picked up with v2.4.4 as last stable release for dp-kafka v2 lib

Reply:
Ok thanks for explaining.